data/Gauravdarkslayer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Gaurav Bhojwani",
+    "type": "personal",
+    "bio": "Keep calm and learn everyday",
+    "avatar": "https://avatars.githubusercontent.com/u/18503497",
```

Comment:
Hey Gaurav, use this link for the avatar instead `https://github.com/Gauravdarkslayer.png`

Reply:
Alright !